src/scss/2021/utilities/_display.scss

```diff
@@ -1,3 +1,7 @@
 .util-height-full {
     height: 100vh;
+}
+
+.util-display-flex {
```

Comment:
hey! did we already have this defined in sparkle classes?

Reply:
It is in Sparkle, but we’re not loading that, and I only needed the one class.